diesel/src/query_builder/functions.rs

```diff
@@ -8,9 +8,16 @@ use super::insert_statement::{Insert, Replace};
 use super::{IncompleteInsertStatement, IncompleteUpdateStatement, IntoUpdateTarget,
             SelectStatement};
 
-/// Creates an update statement. Helpers for updating a single row can be
+/// Creates an `UPDATE` statement. Helpers for updating a single row can be
 /// generated by deriving [`AsChangeset`](query_builder/trait.AsChangeset.html)
 ///
+/// When a table is passed to `update`, every row in the table will be updated.
+/// You can narrow this scope by calling `filter` on the table before passing it in, 
```

Comment:
"You can narrow the scope by calling [`filter`](link to IncompleteUpdateStatement::filter)"